### DISCUSSION TITLE
quasar v1, can noy build/dev , spa and pwa on linux

### DISCUSSION THREAD

**Discussion by treeindark** [quasarframework/quasar#10588] [CLI - PWA mode] [answered]
hello like on title
I start develop on Windows, build and dev ok for spa and pwa.

I migrate on linux, install quasar
pull my repo
finish install

but I cant build or dev build , always: 

`
App · Compiling with Webpack...

 • Compiling:
 └── SPA ████████████████████ 100% done in 31338 ms

 App · :warning:  0 error encountered:

 App · :warning:  [FAIL] Build failed with 0 error. Check log above.
`

No log I don't what to do...

**Comment by pdanpdan** [comment]
Most probably you used upper and lower case letters in the paths and file names and wrote them wrongly, and on windows it worked because it ignores letter case in file names.

**Reply by treeindark** [reply]
I've already coorect path, i 've some errors before,

**Comment by metalsadman** [comment, marked as answer]
most likely a template error, run `quasar build --debug` see if it spit out error logs.

**Reply by treeindark** [reply]
idem

` App · Compiling with Webpack...



 • Compiling:
 └── SPA ████████████████████ 100% done in 36504 ms


 App · ⚠️  0 error encountered:


 App · ⚠️  [FAIL] Build failed with 0 error. Check log above.`

**Reply by treeindark** [reply]
Config template ! thank you metalsadman !
template: `${__dirname}\\src\\index.twig.template.html`

->
template: `${__dirname}/src/index.twig.template.html`
